lib/acts_as_taggable_on/tag.rb

```diff
@@ -21,7 +21,7 @@ def validates_name_uniqueness?
     ### SCOPES:
     scope :most_used, ->(limit = 20) { order('taggings_count desc').limit(limit) }
     scope :least_used, ->(limit = 20) { order('taggings_count asc').limit(limit) }
-    scope :with_category, ->(category , enabled) { where(category: category, enabled: enabled) }
+    scope :with_categories, ->(categories , enabled = true) { where(category: categories, enabled: enabled) }
```

Comment:
@usama-ashraf can you add a space below? There was a space before.

Reply:
@fbrunel below `with_categories ` scope you mean?

Reply:
@usama-ashraf Yeah, between scope and the function below 😬

Reply:
Done 💯